File: app/src/main/java/com/sesac/carematching/transaction/dto/TransactionConfirmDTO.java
package com.sesac.carematching.transaction.dto;

import jakarta.validation.constraints.NotNull;
import lombok.Data;

@Data
public class TransactionConfirmDTO {
    @NotNull
    private String orderId;
    // Nullable (승인 요청에 대한 Response에서만 사용)
    private Integer price;
    // Nullable (KakaoPay에서만 사용)
    private String pgToken;
}

File: src/main/java/com/sesac/carematching/transaction/dto/TransactionConfirmDTO.kt
package com.sesac.carematching.transaction.dto

data class TransactionConfirmDTO (
    val orderId: String,

    // Nullable (승인 요청에 대한 Response에서만 사용)
    val price: Int? = null,

    // Nullable (KakaoPay에서만 사용)
    val pgToken: String? = null
)
